src/routes/jira/jira-sync-post.ts

```diff
@@ -19,7 +22,13 @@ export const JiraSyncPost = async (req: Request, res: Response, next: NextFuncti
 			res.status(404).send("Subscription not found, cannot resync.");
 			return;
 		}
-		await findOrStartSync(subscription, req.log, syncType);
+
+		if (commitsFromDate && commitsFromDate.valueOf() > Date.now()){
+			res.status(400).send("Invalid date, please select historical date!");
```

Comment:
"historical date" doesn't sound right.  Maybe just say "cannot choose a date in the future"?

Reply:
Haha yea reading that now looks odd, 'date does not have enough dinosaurs alive" kinda vibe